WindowsServerDocs/administration/performance-tuning/role/active-directory-server/capacity-planning-for-active-directory-domain-services.md

```diff
@@ -418,14 +419,14 @@ In the next example, the following assumptions are made:
 
 Analyzing the data in the chart (Processor Information(_Total)\% Processor Utility) for each of the DCs:
```

Comment:
@illfated  The underscore before the word, Total, is causing the rest of the text to be italicized. If you would encapsulate this whole string in back ticks and remove the escape character, I'd appreciate it. So, `(Processor Information(_Total)% Processor Utility)`

Reply:
Good point. Thanks for noticing and keeping me informed.